developers/bindings/thing-xml.md

```diff
@@ -509,17 +509,36 @@ The `representation property` will be defined in the thing type XML:
     </thing-type>
 ```
 
-Note that the `representation property` is normally not listed in the `properties` part of the thing type, as this part contains static properties, that are the same for each thing of this thing type.
-The name of the `representation property` identifies a property that is added to the thing in the thing handler upon successful initialization.
+Note that the `representation property` is normally not listed in the `properties` part of the thing type XML, as this part contains static properties, that are the same for all instances of this thing type.
```

Comment:
`...part of the Thing type XML, as this part contains static properties that are the same for all instances of this Thing type.`

Reply:
Ok. Done.